src/styles/blocks/search-bar--small.css

```diff
@@ -0,0 +1,14 @@
+.search-bar--small {
+  height: 42px;
+  background-position: 13px 15px;
+  background-size: 11px;
+}
+
+.search-bar--small .search-bar__input {
+  height: 20px;
+  margin-left: 33px;
```

Comment:
Be consistent with your margins to avoid potential margin collapse. Prefer to add margin on one side (either top or bottom) rather than on both sides.